pkg/config/config_template.yaml

```diff
@@ -3864,100 +3870,100 @@ api_key:
       ## @param ignored_ip_addresses - list of strings - optional
       ## A list of IP addresses to ignore when scanning the network.
       #
-      # ignored_ip_addresses:
-      #   - <IP_ADDRESS_1>
-      #   - <IP_ADDRESS_2>
+      #   ignored_ip_addresses:
+      #     - <IP_ADDRESS_1>
+      #     - <IP_ADDRESS_2>
```

Comment:
This section and the following are part of a list started line 3868, so the indent need to reflect that.